### ISSUE TITLE
Laggy transition between screens on android

### ISSUE THREAD

**Issue by thodwris** [react-navigation/react-navigation#5772]
Hello,

I am struggling with making react-navigation transition smoother between the screens. This only happens on android and not on iOS(at least at iphone X emulator)

This is the configuration of react-navigation on App.js file
```
const SensoriumStackNavigator = createStackNavigator({
  Sensorium: {
    screen: SensoriumScreen,
  },
  Synesthesia: {
    screen: SynesthesiaScreen,
  },
  SynesthesiaItem: {
    screen: SynesthesiaItemScreen,
  },
  MindFulness: {
    screen: MindFulnessScreen,
  },
  BeingAware: {
    screen: BeingAwareScreen,
  },
  Player: {
    screen: PlayerScreen,
    navigationOptions: ({ navigation }) => ({
      headerLeft: null,
      headerRight: <PlayerHeader navigation={navigation} noBanner />,
      headerTransparent: true,
      headerStyle: { height: iPhone5() ? 20 : 40, borderBottomWidth: 1, borderBottomColor: 'transparent' },
    }),
  },
  AudioPlayer: {
    screen: AudioPlayerScreen,
    navigationOptions: ({ navigation }) => ({
      headerLeft: null,
      headerRight: <PlayerHeader audioPlayer navigation={navigation} />,
      headerTransparent: true,
      headerStyle: { height: iPhone5() ? 20 : 40, borderBottomWidth: 1, borderBottomColor: 'transparent' },
    }),
  }
},
  {
    transitionConfig: () => ({
      transitionSpec: {
        duration: 0,
      },
    }),
    defaultNavigationOptions: ({ navigation }) => {
      return {
        headerStyle: { backgroundColor: '#1F1F20', height: iPhone5() ? 30 : 60, borderBottomWidth: 1, borderBottomColor: 'transparent' },
        headerLeft: <MeditateHeader navigation={navigation} />
      }
    }
  });

const UserStackNavigator = createStackNavigator({
  User: {
    screen: UserScreen
  },
  ConfirmUnsubscribe: {
    screen: ConfirmUnsubscribeScreen
  },
  ConfirmMonthlySubscribe: {
    screen: ConfirmMonthlySubscribeScreen
  }
}, {
    defaultNavigationOptions: ({ navigation }) => {
      return {
        headerStyle: { backgroundColor: '#1F1F20', height: 60, borderBottomWidth: 1, borderBottomColor: 'transparent' },
        headerLeft: <MeditateHeader navigation={navigation} />
      }
    }
  });

const PricingStackNavigator = createStackNavigator({
  Pricing: {
    screen: PricingScreen,
    navigationOptions: ({ navigation }) => {
      return {
        headerStyle: { backgroundColor: '#1F1F20', height: 60, borderBottomWidth: 1, borderBottomColor: 'transparent' },
        headerLeft: <MeditateHeader navigation={navigation} />
      }
    }
  },
}, {
    defaultNavigationOptions: ({ navigation }) => {
      return {
        headerLeft: <Icon
          style={{ paddingLeft: 10 }}
          onPress={() => navigation.openDrawer()}
          name="md-menu"
          size={30} />
      }
    }
  });

const ProgressStackNavigator = createStackNavigator({
  Progress: {
    screen: ProgressScreen,
  }
}, {
    defaultNavigationOptions: ({ navigation }) => {
      return {
        headerStyle: { backgroundColor: '#1F1F20', height: 60, borderBottomWidth: 1, borderBottomColor: 'transparent' },
        headerLeft: <MeditateHeader navigation={navigation} />
      }
    }
  });

const SensoriumDrawerNavigator = createDrawerNavigator({
  Sensorium: { screen: SensoriumStackNavigator },
  User: { screen: UserStackNavigator },
  Pricing: { screen: PricingStackNavigator },
  Progress: { screen: ProgressStackNavigator }
}, {
    initialRouteName: 'Sensorium',
    contentComponent: (props) => <SideMenu {...props} />,
    drawerWidth: 300
  });

const AppSwitchNavigator = createSwitchNavigator({
  Sensorium: { screen: SensoriumDrawerNavigator },
  Progress: { screen: ProgressStackNavigator },
  Pricing: { screen: PricingScreen },
  Disclaimer: { screen: DisclaimerScreen }
});

class App extends Component {

  constructor(props) {
    super(props);
    this.state = {
      viewRef: null
    }
  }

  onViewLoaded() {
    this.setState({ viewRef: findNodeHandle(this.viewRef) });
  }

  render() {
    return (
      <Provider store={store}>
        <PersistGate loading={null} persistor={persistor}>
          <View
            style={{ height: '100%', width: '100%', flex: 1 }}
            ref={(viewRef) => { this.viewRef = viewRef; }}
            onLayout={() => { this.onViewLoaded(); }} >
            <BottomBar>
              <AppContainer ref={navigatorRef => {
                NavigationService.setTopLevelNavigator(navigatorRef);
              }}
              />
            </BottomBar>

          </View>
          <ModalContainer />
          <BlurBackground viewRef={this.state.viewRef} {...this.props} />
        </PersistGate>
      </Provider>
    )
  }
}
const AppContainer = createAppContainer(AppSwitchNavigator);
export default App;
```

To prevent some possible answers, I've checked the network tab on debugger and I don't get multiple API requests. The requests happen only once. I also tried to build the apk with release mode as it was mentioned on google as part of solution but with no improvement at all.

Maybe a multiple re-rendering take place somehow but I don't know how to figure it out.

I attach my package.json if it would be helpful

```
"dependencies": {
    "keymirror": "^0.1.1",
    "lodash.isequal": "^4.5.0",
    "react": "16.6.3",
    "react-native": "0.57.8",
    "react-native-blur": "^3.2.2",
    "react-native-fast-image": "^5.2.0",
    "react-native-gesture-handler": "^1.1.0",
    "react-native-linear-gradient": "^2.5.3",
    "react-native-modal": "^7.0.2",
    "react-native-progress-circle": "^2.0.1",
    "react-native-shadow": "^1.2.2",
    "react-native-slider-custom": "git+https://github.com/thodwris/react-native-slider.git",
    "react-native-sound": "^0.10.9",
    "react-native-svg": "^8.0.10",
    "react-native-text-gradient": "0.0.16",
    "react-native-vector-icons": "^6.1.0",
    "react-native-video": "^4.3.1",
    "react-native-youtube": "^1.1.0",
    "react-navigation": "^3.5.1",
    "react-redux": "^6.0.0",
    "redux": "^4.0.1",
    "redux-logger": "^3.0.6",
    "redux-persist": "^5.10.0",
    "redux-saga": "^0.16.2",
    "tipsi-stripe": "^7.4.0"
  }
```

You can see it on the video https://streamable.com/45qsa

Any recommendations?

Thank you

**Comment by vonovak** [comment]
hi there! sorry you're having an issue. as the issue template explains, we require that you provide a runnable example that reproduces the bug on https://snack.expo.io or in a github repository. please try to minimize the superfluous code and focus only on reproducing the bug.

you might wonder, why are we strict about this? the reason is that there are only a small number of maintainers on this project and there are hundreds of people who report issues. if it takes us 20 minutes on average to reproduce each issue and we go through 10 issues each day, that's about half of our day just trying to reproduce bugs, without even beginning to investigate solutions. compare that to 20 minutes for each person who reports an issue.

please create a new issue with this and we'll be happy to review it!
